ruby/classes_and_objects/main.rb

```diff
@@ -0,0 +1,28 @@
+# Define a class Account with three three attributes "name","account_no"
+# & "balance". Name and Balance should be set when creating an object
+# of Account class, account_number should be auto increment. This
+# Account class must have two methods, deposit() and withdraw().
+
+require_relative "lib/account"
+
+p "enter name and balance of 1st customer"
+name_and_initial_balance = gets.chop.split(" ")
```

Comment:
Why is this incorrect?

Reply:
because name of the variable sense is general

Reply:
i think name of variable may be is cutomer_name_and_it's_balance